tools/ShapeChecker/src/main/java/net/open_services/scheck/shapechecker/HttpHandler.java

```diff
@@ -39,11 +43,12 @@
      */
     public HttpHandler()
     {
-         httpClient = HttpClientBuilder
+        Header header = new BasicHeader(HttpHeaders.ACCEPT, "text/html;text/*;*/*");
```

Comment:
Shouldn’t this be a list of RDF MIME types?

Reply:
No, these are the headers to be used when trying to fetch a resource that is not, or might not be, RDF. It turns out that some sites (including jazz.net) do not treat the absence of an ACCEPT header the same way as an ACCEPT header of "*/*", and then fail to redirect to the underlying real page. This was breaking some seeAlso links in OSLC vocabulary checks. Most seeAlso links are to HTML reference pages, hence the preference to get the HTML content type, if available.